scripts/performance/common.py

```diff
@@ -197,7 +202,9 @@ def __runinternal(self, working_directory: str = None) -> tuple:
             ) as proc:
                 if proc.stdout is not None:
                     with proc.stdout:
+                        self.__stdout = StringIO()
                         for line in iter(proc.stdout.readline, ''):
+                            self.__stdout.write(line)
```

Comment:
Is this just so that we can log what is being output from the process we started?

Reply:
This is more so we are able to get out the adb command location from the running of a specific command in runner.py, line 318 for example.

Reply:
This is so that we can capture the adb location from the output of the xharness command.